module/Motion/CrossToMid.h

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file CrossToMid.h
+ * @brief サークルの交点から直線の中点へ移動するクラス
+ * @author YKhm20020
+ */
+
+#ifndef CROSS_TO_MID_H
+#define CROSS_TO_MID_H
+
+#include "InCrossStraight.h"
+#include "InCrossLeft.h"
+#include "InCrossRight.h"
+#include "ColorJudge.h"
+#include "DistanceLineTracing.h"
+
```

Comment:
```suggestion
#include "InCrossStraight.h"
#include "InCrossLeft.h"
#include "InCrossRight.h"
#include "ColorJudge.h"
#include "DistanceLineTracing.h"
#include "BlockAreaMotion.h"

```
今は大丈夫みたいだけど、後々を考えたらあったほうが良いと思う。BlockAreaMotion.getMotionTime()使うときとかに不具合が出るかも。